codec-http/src/main/java/io/netty/handler/codec/http/QueryStringDecoder.java

```diff
@@ -224,6 +280,10 @@ private void decodeParams(String s) {
                     name = decodeComponent(s.substring(pos, i), charset);
                 }
                 pos = i + 1;
+            } else if (withCommaDecoding && c == ',' && name != null) {
```

Comment:
if `withCommaDecoding == true` (or maybe in general?) then does `,` need to be treated more formally as a delimiter? For example what is the expected result of the following:

```java
d = new QueryStringDecoder("?,=f", HttpConstants.DEFAULT_CHARSET, true, 1024, true);
String p = d.path();
Map<String, List<String>> params = d.parameters();
```